### DISCUSSION TITLE
Current Windows Alternatives

### DISCUSSION THREAD

**Discussion by atridadl** [zed-industries/zed#7761] [Feature Requests]
Hi y'all! Instead of the 1000th post asking about the status of windows, I'm curious: **what do you guys use when you need to be on windows?** I have a MacBook and a windows desktop and sometimes I work on my desktop. VSCode is a bit bloated for my taste lately. Any suggestions?

**Comment by SHADOWELITE7** [comment]
Just discovered Zed today and noticed on its website when looking at the speed graph. Would recommend [Sublime Text](https://www.sublimetext.com/). I have been using it for 6 years. It was recommended to me by an online acquaintance who used Linux and didn't like the bloat in programs made in Electron like Atom & VSCode. Even though its a text editor with lots of packages to choose from. If you like IDE's. There is a group that adds language servers for Sublime Text called [SublimeLSP](https://lsp.sublimetext.io/). I like it more than VSCode because you can curate what language servers you are going to use in your current project. 

![image](https://github.com/zed-industries/zed/assets/33847580/870f6a51-99b6-4945-ae68-301ab23b0bad)
![image](https://github.com/zed-industries/zed/assets/33847580/d970ed7f-87f4-4638-8db1-45c79370a931)
![image](https://github.com/zed-industries/zed/assets/33847580/efb96d0b-2126-4916-948c-23d280fed9ea)

**Reply by atridadl** [reply]
Yeah I've been using Sublime Text lately too and it's grown on me. That and sublime merge. Both run extremely well and are nice to use.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Comment by vanyastaff** [comment]
fine. but i think to go other app if zed not fix memory leak.. 90GB+
